monitoring/api/MonitoringSample/README.md

```diff
@@ -18,18 +18,8 @@ can use [this older sample](
     [Click here](https://console.cloud.google.com/flows/enableapi?apiid=monitoring.googleapis.com&showconfirmation=true)

```

Comment:
Update markdown href path to root README.md file:
    [root README](../../README.md)
to be
    [root README](../../../README.md)